docs/_posts/2016-01-12-developers-manual.md

```diff
@@ -408,16 +408,22 @@ Pull the tag or snapshot that you want to build and package it:
 
     export version="0.2.35-SNAPSHOT"
     git checkout "tags/${version}"
-    mvn -P external clean install package
-    docker build -t "misolims/miso-lims:${version}" --build-arg version="${version}" --no-cache .
+    docker build --target flyway-migrate -t "misolims/miso-lims-migration:${version}" --no-cache .
+    docker build --target webapp -t "misolims/miso-lims-webapp:${version}" --no-cache .
+    docker tag "misolims/miso-lims-migration:${version}" misolims/miso-lims-migration:latest
+    docker tag "misolims/miso-lims-webapp:${version}" misolims/miso-lims-webapp:latest
 
-Once the build completes, test it by launching it:
+Once the build completes, test it by launching it. This command uses the default 
+environment variables in .env and relies on `.miso_db_password` file existing.
 
-    docker run -p 8090:8080 --name "miso${version}" -t "misolims/miso-lims:${version}"
+    export MISO_TAG="0.2.35-SNAPSHOT" && docker-compose up
```

Comment:
Should MISO_TAG be `${version}`?

Reply:
I had it as version but it's actually the tag on the docker container. As long as we're happy keeping the tags and versions in sync, I can substitute it.

Reply:
Oh never mind, you mean the actual value. I can change that.

Reply:
Fixed